### ISSUE TITLE
I can't resubmit my assessment (researcher to admin)

### ISSUE THREAD

**Issue by gjacovella** [NRGI/rgi-assessment-tool#219]
I'm stuck with the assessment on Italy 2017: the admin can review the questions but cannot approve them. The researcher has marked all the questions as complete but cannot resubmit the questionnaire to the admin.

**Comment by ahasemann** [comment]
This is addressed in #195

**Comment by byndcivilization** [comment]
just caught the same bug in last code. will push right now.

**Comment by gjacovella** [comment]
@byndcivilization This morning the peer reviewer couldn't submit the trial questions, so I saved the answers and I noticed that the admin could already choose the final score. Then when I tried to assign the full assessment to the peer reviewer, I couldn't (I guess because the peer reviewer didn't submit the trial questions to the admin, so the admin couldn't assign the assessment). However, the peer reviewer did receive the assessment anyway. How is that possible? 
![image](https://cloud.githubusercontent.com/assets/16592996/13424674/25380582-df9a-11e5-9095-1faaa170a5a9.png)
